bin/validate_lastnight_fba

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# requires source /project/projectdirs/desi/software/desi_environment.sh master
+from astropy.table import Table
+import numpy as np
+import os
+import argparse
+import sys
+
+parser = argparse.ArgumentParser()
+#required arguments
+parser.add_argument("--night", help="use this to specify the night, this is required unless you want 20210921 for some reason",default='20210921')
+#not required arguments
+parser.add_argument("--specprod",help="set specprod to something other than daily",default='daily')
+parser.add_argument("--outroot",help="the root directory for temporary outputs",default=os.environ['CSCRATCH'])
+args = parser.parse_args()
+
+month = args.night[:6]
+
+if args.night == '20210921':
+    print('DID YOU REALLY MEAN TO RUN THIS FOR '+args.night)
+    
+if not os.path.exists(args.outroot):
+    
+    print('the root directory '+args.outroot+' for where temporary output is meant to go does not exist; provide a different --outroot argument')
+    sys.exit('the root directory '+args.outroot+' for where temporary output is meant to go does not exist; provide a different --outroot argument; exiting')
+    
+
+desiroot = os.path.expandvars('$DESI_ROOT/')
```

Comment:
Having previously suggested expandvars, I have now discovered that it is less clever than I thought.  If $DESI_ROOT isn't defined, that evaluates to the string "$DESI_ROOT/", yuck.  I think we need this to be:
```python
if 'DESI_ROOT' in os.environ:
    desiroot = os.environ['DESI_ROOT']+'/'
else:
    # print some error message and exit
```